database/migrations/2021_11_23_062046_create_customers_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateCostumersTable extends Migration
```

Comment:
nama filenya juga create_customers_table
```suggestion
class CreateCustomersTable extends Migration
```